src/main/java/com/adyen/service/TerminalCloudAPI.java

```diff
@@ -83,4 +82,22 @@ public TerminalAPIResponse sync(TerminalAPIRequest terminalAPIRequest) throws IO
         }.getType());
         return terminalAPIResponse;
     }
+
+    /**
+     * Checks if SaleToAcquirerData is set. If it is not creates an encoded base 64 string with application info.
+     *
+     * @param terminalAPIRequest TerminalAPIRequest
+     * @return TerminalAPIRequest
+     */
+    private static TerminalAPIRequest getApplicationInfo(TerminalAPIRequest terminalAPIRequest) {
```

Comment:
method name is not appropriate according to operation. it is setting saletoacquiredata and not getting applicationinfo.